src/main/java/ch/uzh/ifi/hase/soprafs22/game/SurrenderInfo.java

```diff
@@ -1,13 +1,22 @@
 package ch.uzh.ifi.hase.soprafs22.game;
 
+import java.util.List;
+import java.util.Map;
+
 public class SurrenderInfo {
     private final long surrenderedPlayer;
+    private final Map<Long, List<Integer>> rankedScoreDeltas;
 
-    public SurrenderInfo(long surrenderedPlayer) {
+    public SurrenderInfo(long surrenderedPlayer, Map<Long, List<Integer>> rankedScoreDeltas) {
```

Comment:
I don't understand why these ranked deltas need to be in SurrenderInfo instead of GameOver.